src/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/CompilerServices/IOUtils.vb

```diff
@@ -0,0 +1,152 @@
+' Licensed to the .NET Foundation under one or more agreements.
+' The .NET Foundation licenses this file to you under the MIT license.
+' See the LICENSE file in the project root for more information.
+
+Imports System
+Imports System.Security
+Imports System.Security.Permissions
+Imports System.IO
+Imports System.Text
+
+Imports Microsoft.VisualBasic.CompilerServices.ExceptionUtils
+Imports Microsoft.VisualBasic.CompilerServices.Utils
+
+Namespace Microsoft.VisualBasic.CompilerServices
+
+    <System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)> _
+    Class IOUtils
+        ' Prevent creation.
+        Private Sub New()
+        End Sub
+
+        <SecurityCritical()> _
```

Comment:
You can remove `Imports System.Security.Permissions` everywhere and `SecurityCritical` etc